packages/api/src/api/helix/charity/HelixCharityCampaignAmount.ts

```diff
@@ -0,0 +1,35 @@
+import { DataObject, rawDataSymbol } from '@twurple/common';
+
+/** @private */
+export interface HelixCharityCampaignAmountData {
+	value: number;
+	decimal_places: number;
+	currency: string;
+}
+
+export class HelixCharityCampaignAmount extends DataObject<HelixCharityCampaignAmountData> {
+	/**
+	 * The monetary amount. The amount is specified in the currency’s minor unit.
+	 * For example, the minor units for USD is cents, so if the amount is $5.50 USD, `value` is set to 550.
+	 */
+	get value(): number {
+		return this[rawDataSymbol].value;
+	}
+
+	/**
+	 * The number of decimal places used by the currency. For example, USD uses two decimal places.
+	 * Use this number to translate `value` from minor units to major units by using the formula:
+	 *
+	 * `value / 10^decimalPlaces`
```

Comment:
I think this would make sense to implement as an additional getter, think `localizedValue` or something like that (should still be a number, not a string, so number formatting such as comma rules can be implemented at userland)